projects/web/package.json

```diff
@@ -18,6 +18,8 @@
     "auth0-js": "^9.10.0",
     "digital-link.js": "^1.0.0",
     "jsqr": "^1.2.0",
+    "moment": "^2.24.0",
+    "moment-timezone": "^0.5.25",
```

Comment:
Did you check how this affected the bundle size? IIRC the timezone library is quite large and may not be required?

Reply:
Removed